README.md

```diff
@@ -50,6 +50,10 @@ folder and try to start the server again)_
 
 #### Redeploy smart-contracts and generate mock data
 
+TODO: update this section to use truffle and remove redeploy function from clojure
+
+TODO: to run deploy on testnet: `truffle migrate --network ropsten`
```

Comment:
There's Deploying Name Bazaar smart contracts section below, you can add a Truffle deploy subsection there as the first option, but keep the clojure one as fallback.

Reply:
You are commenting on outdated files (don't know how :D)

Reply:
I forgot I only clicked the first commit :D